packages/server/api/src/app/database/seeds/openops-delete-old-opportunities-table.ts

```diff
@@ -41,28 +41,20 @@ export const deleteOldOpportunitiesTable = async (): Promise<void> => {
   try {
     const { token } = await authenticateDefaultUserInOpenOpsTables();
 
-    const opportunitiesTableIdOld = await getTableIdByTableName(
```

Comment:
i added a new function so we dont have the ugly try catch, was unreadable anyways